app/src/main/java/org/simple/clinic/summary/addphone/MissingPhoneReminder.kt

```diff
@@ -0,0 +1,28 @@
+package org.simple.clinic.summary.addphone
+
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import io.reactivex.Flowable
+import org.simple.clinic.patient.PatientUuid
+
+@Entity(tableName = "MissingPhoneReminder")
+data class MissingPhoneReminder(
+    @PrimaryKey
+    val patientUuid: PatientUuid,
+    val reminded: Boolean
```

Comment:
Even if we do this, we won't need to change the repository API signature since it doesn't actually read this property and controllers will remain untouched.

Reply:
> I can see us wanting to change from "Never show this dialog once its shown" to something like "Show this dialog once every X days" and this will automatically support both cases.

Sure